src/odemis/acq/stream/_projection.py

```diff
@@ -713,13 +714,20 @@ def _project2RGBPolar(self, ebeam_pos, pol_pos, cache_raw):
                 else:
                     colormap = "seismic"
 
+                # data including normalization need cropping of the pole, to ensure proper scaling
+                if pol_pos in [MD_POL_UP, MD_POL_DOP, MD_POL_DOCP, MD_POL_DOLP]:  # TODO: hardcoded for now
```

Comment:
I'm not it's worthy a "TODO". Do you think there is a better way to do it, but it'd too long/hard to do it now? I think that it's pretty much the best way we can do it, given the fact we want to minimize the user options.

Reply:
True. Thought about having a global variable similar to POL_POSITIONS_RESULTS, but just for those, but not sure that is feasible. Removed the TODOs